import_module/run_import.py

```diff
@@ -11,15 +11,15 @@
 
 import logging
 import os
-import time
```

Comment:
So, it looks like you removed this import but it's used around this line, https://github.com/mapswipe/python-mapswipe-workers/pull/24/files#diff-37f15fe964b547fd83d5ac92f5118804R482